src/libraries/Microsoft.Extensions.Hosting.Abstractions/src/IHostApplicationBuilder.cs

```diff
@@ -0,0 +1,64 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Extensions.Hosting
+{
+    /// <summary>
+    /// A builder for hosted applications and services which helps manage configuration, logging, lifetime, and more.
+    /// </summary>
+    public interface IHostApplicationBuilder
+    {
+        /// <summary>
+        /// A central location for sharing state between components during the host building process.
```

Comment:
```suggestion
        /// Gets a central location for sharing state between components during the host building process.
```
Same for other properties.